gulp-tasks/convert-design-tokens.js

```diff
@@ -53,7 +53,9 @@ const convertDesignTokens = (cb) => {
 
   tokens.colors.forEach((group) => {
     group.items.forEach((color) => {
-      result += `'${nameSlug(group.group + '-' + color.name)}': ${color.hex},`;
+      result += `'${nameSlug(group.group + '-' + color.name)}': ${
+        color.value.includes('data') ? `url("${color.value}")` : color.value
```

Comment:
Surely these aren't all colors now (it's the section for colors / `token.color`) if a bunch of them are actually SVG backgrounds?

Also, can you change`.includes` => `.startsWith("data:")`) ?